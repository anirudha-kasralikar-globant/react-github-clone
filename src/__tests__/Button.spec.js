// @flow
import React from 'react';
import { create } from 'react-test-renderer';

function Button(props) {
  return (
    <button type="button" className="abcd">
      Nothing to do for now
    </button>
  );
}

describe('Button component', () => {
  //
  test('Matches the snapshot', () => {
    const button = create(<Button />);
    expect(button.toJSON()).toMatchSnapshot();
  });
  //
});
