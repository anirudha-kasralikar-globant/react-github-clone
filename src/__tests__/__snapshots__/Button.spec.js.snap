// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component Matches the snapshot 1`] = `
<button
  className="abcd"
  type="button"
>
  Nothing to do for now
</button>
`;
